perfdash/config.go

```diff
@@ -179,16 +179,28 @@ var (
 			}},
 		},
 		"Scheduler": {
-			"SchedulingLatency": []TestDescription{{
-				Name:             "density",
-				OutputFilePrefix: "SchedulingMetrics",
-				Parser:           parseSchedulingLatency,
-			}},
-			"SchedulingThroughput": []TestDescription{{
-				Name:             "density",
-				OutputFilePrefix: "SchedulingThroughput",
-				Parser:           parseSchedulingThroughputCL,
-			}},
+			"SchedulingLatency": []TestDescription{
+				{
+					Name:             "density_pod-anti-affinity",
```

Comment:
`density_*` items need to be before the `density` item because of how data file prefixes work.

Reply:
Nice finding! Would you mind documenting it in a comment?

Reply:
Done.